app/forms/idv/api_image_upload_form.rb

```diff
@@ -120,13 +122,25 @@ def validate_pii_from_doc(client_response)
     end
 
     def extra_attributes
-      @extra_attributes ||= {
+      return @extra_attributes if defined?(@extra_attributes)
+      @extra_attributes = {
         attempts: attempts,
         remaining_attempts: remaining_attempts,
         user_id: user_uuid,
         pii_like_keypaths: [[:pii]],
         flow_path: params[:flow_path],
       }
+
+      if front_image_bytes
+        @extra_attributes[:front_image_hash] =
```

Comment:
1. could we also update the `analytics_events` for the event that these go to? I think `idv_doc_auth_submitted_image_upload_vendor` maybe?
2. WDYT about calling it `_digest` or `_sha` instead of `_hash`? I thought we sent a `phone_digest` in some analytics event but I can't find it, so I guess not

Reply:
it's `phone_fingerprint`, which I think would be fine to use here as well?

Reply:
yes! `_fingerprint`! big fan

Reply:
Cool, makes sense to me, renamed. I also added the documentation to `analytics_events`